src/Console/Command/HelpCommand.php

```diff
@@ -50,14 +50,16 @@ public static function toString($value): string
 
     /**
      * Returns the allowed values of the given option that can be converted to a string.
+     *
+     * @return list<null|AllowedValueSubset|scalar>
```

Comment:
but... method prototype says that it can return null?

Reply:
Sorry, that's a mistake (ref #6584).